src/services/transfer/transferService.ts

```
@@ -162,8 +162,8 @@ export async function createTransfer(
   // prevent precision loss when amount has up to 7 decimal places.
   const [wholePart, fracPart = ""] = amount.split(".");
   const amountInSmallestUnit =
-    parseInt(wholePart, 10) * 100 +
-    parseInt(fracPart.slice(0, 2).padEnd(2, "0"), 10);
+    parseInt(wholePart, 10) * 10000000 +
+    parseInt(fracPart.slice(0, 7).padEnd(7, "0"), 10);
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)